### ISSUE TITLE
Remove the ChromeOS User Agent

### ISSUE THREAD

**Issue by hmlendea** [hmlendea/geforcenow-electron#17]
Some time ago a GamingOnLinux article reported that GFN works well with Linux without spoofing the User Agent string to a ChromeOS one.
Since then that continued to be the case and it seems like the spoofed UA is not necessary anymore.

If all goes well, it should be removed in the next version.

**Comment by Siedhuss** [comment]
for information
other user agent strings could result in no choice of keyboard loayout inside gfn

**Comment by TheRealFanjin** [comment]
I'm on Linux and having no user agent switcher just said that my device was unsupported

**Comment by hmlendea** [comment]
> I'm on Linux and having no user agent switcher just said that my device was unsupported

Huh, interesting. What browser did you use?

**Comment by TheRealFanjin** [comment]
Both Chrome and Edge
and maybe Firefox too

**Comment by tosziro** [comment]
I can confirm that GeForce Now works correct with Brave and Chromium browser without any workarounds. We dont need this ChromeOS user agent any more.

**Comment by hmlendea** [comment]
```
Soon we’ll bring that experience to more Chrome platforms, including Linux*, PC, Mac and Android. Stay tuned for updates as we approach a full launch early next year.
```

From my understanding it's not working for everyone still, as that is yet to happen next year. For some it has been working for a while (it did for me since before I made this PR), but it's not yet safe to remove it for the masses.

Once the official announcement is made from Nvidia, I will merge this PR

**Comment by hmlendea** [comment]
Merged in #36
